conans/test/unittests/util/tools_test.py

```diff
@@ -876,6 +876,31 @@ def build(self):
         client.save({"conanfile.py": conanfile, "file.txt": "hello\r\n"})
         client.run("create . user/channel")
 
+    @attr('slow')
+    @pytest.mark.slow
+    @attr('local_bottle')
```

Comment:
A test with a local bottle running shouldn't be necessary to test this utility, this mark can be removed